.github/CONTRIBUTING.md

```diff
@@ -37,8 +32,7 @@ Contributing to Issues
 * If the issue is marked as [`Up-for-Grabs`][up-for-grabs],
   the PowerShell Maintainers are looking for help with the issue.
```

Comment:
I think we've been using Up-for-grabs mostly to indicate easy/small/non-critical issues as a way to start contributing code and not really as needing help